test/ares-test.cc

```diff
@@ -385,7 +385,7 @@ void MockServer::ProcessFD(int fd) {
      * need to split */
     while (tcp_data_len_ > 2) {
       int tcplen = (tcp_data_[0] << 8) + tcp_data_[1];
-      if (tcp_data_len_ - 2 < tcplen)
+      if (tcp_data_len_ - 2 < (size_t)tcplen)
         break;
```

Comment:
maybe we should make tcplen size_t in the first place on line up?

Reply:
yes, that's indeed nicer